### ISSUE TITLE
dataset  can't download 

### ISSUE THREAD

**Issue by YuLengChuanJiang** [yabufarha/ms-tcn#20]
Hello, when I download several Gs of data, it always stops downloading automatically. Can you give me some advice?

**Comment by yabufarha** [comment]
Hi,
Try to use the alternative link provided in the readme file.
https://zenodo.org/record/3625992#.Xl9A5ahKhPZ

**Comment by YuLengChuanJiang** [comment]
yes, I have used the alternative link, only can download several Gs of data. Are there any better solutions?

**Comment by yabufarha** [comment]
Are you sure you tried this link https://zenodo.org/record/3625992#.Xl9A5ahKhPZ ??
Can you please try again and let me know if it does not work.

**Comment by YuLengChuanJiang** [comment]
![sendpix0](https://user-images.githubusercontent.com/43591197/75850282-e0ff9780-5e21-11ea-9e84-b41a4c57ec6c.jpg)
At the beginning, the network speed can reach several M per second, and then the download will stop, just like the picture. There's no problem with my network.

**Comment by yabufarha** [comment]
I'll check the problem.

**Comment by YuLengChuanJiang** [comment]
After checking, please tell me. I am very interested in your work.

**Comment by yabufarha** [comment]
The link is working. I've just downloaded the data and everything is fine.
You should be able to download the data. Please check whether you have other problems.

**Comment by YuLengChuanJiang** [comment]
Ok,thanks a lot!

**Comment by morgancjs** [comment]
> Ok,thanks a lot!

Hello! I meet the same problem to fail to download the data from the second link. It ends and restarts automatically. Have you succeeded and what can I do? Thank you!

**Comment by guuzaa** [comment]
Hi,
What kind of I3D features have you extracted? Both optical flow and RGB ? 

> Hi,
> Try to use the alternative link provided in the readme file.
> https://zenodo.org/record/3625992#.Xl9A5ahKhPZ

**Comment by yabufarha** [comment]
Hi @guuzaa,
Yes. We used both RGB and flow streams.

**Comment by yabufarha** [comment]
@YuLengChuanJiang @morgancjs 
Did you mange to download the data? Did you figure out what is the problem?

**Comment by YuLengChuanJiang** [comment]
@yabufarha Yes, I did. I successfully downloaded the data through the school‘s network.  However, I don't know the reason why the previous download failed.

**Comment by morgancjs** [comment]
I tried once more just now, but it failed again at the point that 6-7G has been download. I'm sorry I don't know why. Maybe my VPN doesn't work well, but I have downloaded extracted feature for action segmentation successfully several months ago from other github link. Thank you!






------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "yabufarha/ms-tcn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年5月27日(星期四) 下午4:01
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [yabufarha/ms-tcn] dataset  can't download  (#20)





 
@YuLengChuanJiang @morgancjs
 Did you mange to download the data? Did you figure out what is the problem?

**Comment by guuzaa** [comment]
Hi @yabufarha ,
Thanks for your quickly reply. 
There is one more question. I have checked your dataset, I wonder how did you extract features , and made these dimensions same as the number of the frames of corresponding videos.  Could you share with us your approach? 
Thanks again :)

**Comment by yabufarha** [comment]
Dear @guuzaa, 
The code we used for extracting the features is already publicly available https://github.com/ahsaniqbal/Kinetics-FeatureExtractor

**Comment by guuzaa** [comment]
@yabufarha Thanks for your reply, I will check it.

**Comment by guuzaa** [comment]
> @yabufarha Yes, I did. I successfully downloaded the data through the school‘s network. However, I don't know the reason why the previous download failed.

Hi @YuLengChuanJiang ,
I've tried and download several times from this link https://zenodo.org/record/3625992#.Xl9A5ahKhPZ , unfortunately it doesn't work like you. Can you share the dataset you downloaded with me through BaiduYun?   Thanks anyway.

**Comment by guuzaa** [comment]
> > @yabufarha Yes, I did. I successfully downloaded the data through the school‘s network. However, I don't know the reason why the previous download failed.
> 
> Hi @YuLengChuanJiang ,
> I've tried and download several times from this link https://zenodo.org/record/3625992#.Xl9A5ahKhPZ , unfortunately it doesn't work like you. Can you share the dataset you downloaded with me through BaiduYun? Thanks anyway.

Hi @YuLengChuanJiang,
I've downloaded the feature successfully. Sorry to bother you. ❤️

**Comment by wanduoz** [comment]
Hi @guuzaa , Can you share the dataset you downloaded with me through BaiduYun? I can't download the whole dataset due to network problem. Thank you very much.

**Comment by YuLengChuanJiang** [comment]
您好！您的邮件我已收到，我会尽快查看并回复 ，请不用担心。 祝您万事如意！     
                                                                   冷川江

**Comment by 287959707** [comment]
Hello, do you have 50 salads dataset? (Original file video and timestamps zip)
Due to the official dataset website https://cvip.computing.dundee.ac.uk/ Unable to access.
May I ask if you can share it with me? Thank you very much.

**Comment by YuLengChuanJiang** [comment]
您好！您的邮件我已收到，我会尽快查看并回复 ，请不用担心。 祝您万事如意！     
                                                                   冷川江
